Gemfile.lock

```diff
@@ -331,7 +334,7 @@ GEM
       psych (~> 3.1.0)
       rainbow (>= 2.0, < 4.0)
     require_all (3.0.0)
-    responders (3.0.0)
+    responders (3.0.1)
```

Comment:
### [responders](https://github.com/heartcombo/responders)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.0.0 → 3.0.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **16** commits. See the full changes on [the compare page](https://github.com/heartcombo/responders/compare/v3.0.0...v3.0.1).

These are the first 10 commits:
- (0b639ec) [Simplify README description](https://github.com/heartcombo/responders/commit/0b639ec3c5fd028535a83b292ce2f756c8f468c2)
- (f3b2b3a) [Merge pull request #212 from deivid-rodriguez/simplify_readm…](https://github.com/heartcombo/responders/commit/f3b2b3acbbbda16188831c841f2b8c1b8ba31d1e)
- (80d1551) [Use released rails 6 to test on travis](https://github.com/heartcombo/responders/commit/80d1551c76d1d5fdae0712bba4ff83a5dbccd37f)
- (3b9e0fa) [Add rails head and ruby head to CI](https://github.com/heartcombo/responders/commit/3b9e0fab1e2a4a7d46ca133958e81ae35254da72)
- (a7c3779) [Merge pull request #215 from plataformatec/mf-add-rails-head…](https://github.com/heartcombo/responders/commit/a7c37798f860b2be7bc1107f3a9dc4a3f5d307a6)
- (097e440) [Allow failures against rails head in CI](https://github.com/heartcombo/responders/commit/097e4404507d3c06ed84a7cded31145cfcdb38af)
- (29faa05) [Merge pull request #216 from plataformatec/mf-allow-failures…](https://github.com/heartcombo/responders/commit/29faa05fb2ccf618484195ce5ece320c4b95477a)
- (7256bed) [Minor README improvements](https://github.com/heartcombo/responders/commit/7256bed4133bdddc489519ee64d49eb88a488f92)
- (f7f381f) [Merge pull request #219 from martinbjeldbak/patch-1](https://github.com/heartcombo/responders/commit/f7f381f7f0d30dee5cc00ba22bf386b00698c21a)
- (c51c249) [Fix typo in 'respond_with' documentation](https://github.com/heartcombo/responders/commit/c51c24934659be1a160a4db66a661460e1dfdc6d)


</details>